.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   GO_VERSION: '1.22'
-  ACTION_MSRV_TOOLCHAIN: 1.66.1
+  ACTION_MSRV_TOOLCHAIN: 1.67.0
```

Comment:
We need to clarify with downstream if we can do that. I doubt looking at the latest conmon-rs builds in brew.

Reply:
No worries. I moved my local environment to v1.66.1, which resolves the time crate build issue. too.

Reply:
For posterity, the following was the time crate build issue:

![Screenshot from 2024-07-30 16-54-37](https://github.com/user-attachments/assets/74741a39-55a0-48eb-84cd-9d58aeb0f493)

Reply:
Actually, I might have spoken too soon. We seem to have the same issue too, per:

https://github.com/containers/conmon-rs/actions/runs/10164569684/job/28110266502?pr=2317#step:6:139

Reply:
I don't seem to have this issue locally any more.

Reply:
OK. We might need to pin down Rust version for the static build job too, per:

```console
rustc 1.79.0 (129f3b996 2024-06-10)
/home/runner/.cargo/bin/rustup -V
rustup 1.27.1 (54dd3d00f [20](https://github.com/containers/conmon-rs/actions/runs/10164569684/job/28110266502?pr=2317#step:5:21)24-04-24)
```

For reference:

- https://github.com/containers/conmon-rs/actions/runs/10164569684/job/28110266502?pr=2317#step:5:20

Reply:
Or maybe I broke it when I removed sudo... let me check.

Reply:
No. Wasn't me. :sweat_smile:

Reply:
@saschagrunert, the RPM builds on a more modern Fedora version have the same issue.

How do you want to go about it? Since time crate upgrade is a no-no? (requires slightly newer Rust)

Reply:
I was thinking about this yesterday. We could either fork the crate and patch it, see: https://github.com/saschagrunert/time/tree/v0.3.23-fix

But Cargo is a bit picky on versions, so we would have to replace it for all dependencies, not just the direct ones.

Reply:
Probably like this: https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#using-patch-with-multiple-versions